### ISSUE TITLE
is This normal when running it

### ISSUE THREAD

**Issue by yibbee** [CISOfy/lynis#774]
I was hoping for confirmation about this as I am worried it might be coming from my PC & not from running Lynis. But when I run lynis from console it gets to the Ports and packages part of the scan & then stops scanning when a popUp of the game PacMan in console form appears & will not finish scanning until I close the PacMan game.
I am running linux , is there any chance at all its part of Lynis & if not, then I do not think it would be worth reporting to Lynis Security mailing as it could be a malware attack?
Thanks

**Comment by yibbee** [comment]
Sorry I just found out what is going on" ,I noticed that when it got to Ports & Pkgs part of the scan I noticed it says (Searching pacman package manager                        [ FOUND ]
- Querying pacman package manager.
Only problem is I do not use pacman package manager as I am using RPM based distro.
So I searched for pkgs installed to see if I had installed a pacman game in the past I may have forgot about & Sure enough its installed & its mistaking it for Arch Pacman manager.
Wow'" I was really worried . But also really thought the lynis developers might of added it so that people could play pacman while waiting for the scan to finish as I have seen similar tactics used in linux in the earlier days but I had doubts.
Thanks anyWay
